### DISCUSSION TITLE
Welcome to ClientBase Discussions!

### DISCUSSION THREAD

**Discussion by zihasz** [ZWareDevelopment/ClientBase#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MOMIN5** [comment]
Do you ever plan to port it to Fabric 1.16?

**Reply by zihasz** [reply]
Yes, when most mainstream servers update to 1.16 I will update too. Also I'm not too sure if I will use Fabric or Forge.
